.github/workflows/cypress.yml

```
@@ -75,7 +75,7 @@ jobs:
       matrix:
         # Run multiple copies of the current job in parallel
         # Please increase the number or runners as your tests suite grows
-        containers: [1, 2, 3]
+        containers: ["component", 1, 2, 3]
```

```suggestion
        containers: [1, 2, 3]
```
We may need to cancel this change, not sure, let’s see what CI says